src/Chapter2.hs

```diff
@@ -841,7 +882,8 @@ list.
 
 🕯 HINT: Use the 'cycle' function
 -}
-rotate = error "rotate: Not implemented!"
+rotate :: Int -> [Int] -> [Int]
+rotate n l = take (length l) (drop n (cycle l))
```

Comment:
It looks like this function doesn't return empty lists on negative numbers. Also, it does a little bit more than work when the given number is big and the list is small. You will cycle through the list a lot. Maybe you can think of a way to optimize this :slightly_smiling_face: